### PR TITLE
Signup changes

### PR DESCRIPTION
**Pull request by sangam12-08**

### CODE REVIEW

File: src/app/login/login.component.css [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  margin: 0;
+}
+
+.center{
+  /* t-align: center;tex */
+  height:70em;
+  display:flex;
+  justify-content: center;
+  align-items: center;
+  background-image: url("../register/image.jpg");
```
Comment (sassyMatter) [inline src/app/login/login.component.css]: @sangam12-08  let's keep images under asset folder only

File: src/app/login/login.component.html [hunk 2]
```diff
@@ -1,22 +1,23 @@
-<div class="col-md-12">
+<div class="col-md-12 center">
     <div class="card card-container">
       <img
         id="profile-img"
-        src="//ssl.gstatic.com/accounts/ui/avatar_2x.png"
+        src="https://www.w3schools.com/w3images/avatar2.png"
```
Comment (sassyMatter) [inline src/app/login/login.component.html]: can we just download the source image and keep it in assets @sangam12-08 ?

File: src/app/register/register.component.html [hunk 3]
```diff
@@ -53,10 +58,10 @@
           </div>
         </div>
         <div class="form-group">
-          <label for="password">Password</label>
+          <label class="lab_password" for="password">Password</label>
           <input
             type="password"
-            class="form-control"
+            class="form-control inpat"
```
Comment (sassyMatter) [inline src/app/register/register.component.html]: I see you could not keep it "input" , we can name it input-password or something, just more readable
@sangam12-08

**Review by sassyMatter — APPROVED** [review]

**Comment by Harshit8-Singh** [comment]
.

**Pull request closed without merging**